src/main/java/org/sasanlabs/internal/utility/SecureConstants.java

```diff
@@ -0,0 +1,30 @@
+package org.sasanlabs.internal.utility;
+
+/**
+ * Represents the Level of the Vulnerability Type that is considered secure.
+ */
+public interface SecureConstants {
```

Comment:
As discussed, we don't need SecureConstants as we are relying on Level and Variant(Type of Secure/Unsecure)

Reply:
Please take a look at https://github.com/SasanLabs/VulnerableApp/pull/278#discussion_r623483099 and https://github.com/SasanLabs/VulnerableApp/pull/278#discussion_r623490100

Reply:
please have a look at https://github.com/SasanLabs/VulnerableApp/pull/278/files#r623531869

Reply:
Resolved in 3211f26